.github/workflows/_integration_tests.yaml

```diff
@@ -64,6 +64,12 @@ jobs:
           - name: dbless-combined-services
             test: dbless
             feature_gates: "GatewayAlpha=true,CombinedServices=true"
+          - name: dbless-fill-ids
+            test: dbless
+            feature_gates: "GatewayAlpha=true,FillIDs=true"
+          - name: postgres-fill-ids
+            test: postgres
+            feature_gates: "GatewayAlpha=true,FillIDs=true"
```

Comment:
Our tests are going to take ages 🕐 🤷 :)

Reply:
Do you think we should not add these tests?

Reply:
I'm not staying we shouldn't add these but a  revamp of our CI is long overdue